doc/ProjectSetUp.md

```diff
@@ -0,0 +1,24 @@
+# Project Set-Up/Testing
+
+This section will go through: the basic prerequisites needed to develop for PowerPointLabs, how to set up the development environment and how to run tests within the development environment.
+
+## Dev Prerequisites
+1. Install Office 2016 or 2013 with PowerPoint
+1. Install SourceTree (recommended), GitHub for Windows, or at least, Git
+1. Install Visual Studio 2015
```

Comment:
@junkiattan, @leeyh20 
Correct me if I'm wrong but I think you guys use 2017. Maybe we can suggest that instead? I personally was wondering what the difference was when I first started contributing.

Reply:
Yes I use a mixture of 2017 and 2015. I'm not sure of the differences!

Reply:
Noted and changed